### ISSUE TITLE
Replace "Are you lost?" with purple subscription prompt

### ISSUE THREAD

**Issue by dmnyc** [damus-io/damus#2202]
It seems like this screen could be better utilized to provide more useful features and content to Damus, such as:

- Advanced search
- Onboarding tutorials
- Nostr resources
- Damus Purple signup link
- Sponsored content
etc.

![image](https://github.com/damus-io/damus/assets/3638467/6ca8e752-91ba-4951-b586-8ede4a682a7d)

**Comment by karnagebitcoin** [comment]
Even something as simple as a "did you know..." and refresh it every time to teach some concepts.

**Comment by alltheseas** [comment]
Love this. Will discuss with team

**Comment by alltheseas** [comment]
![image](https://github.com/damus-io/damus/assets/64376233/8a9de3dd-52ca-420a-9727-70aeb039b606)

**Comment by alltheseas** [comment]
Add quote bar to damoose

**Comment by alltheseas** [comment]
https://www.rawpixel.com/image/6334651/png-sticker-public-domain

**Comment by vrod1313** [comment]
I think the one Will did is probably better for this case....this is one of the other ones for merch potentially. https://image.nostr.build/546215b623e9b7f9c90b514f26f3b4c0df04c764b18ea1b365b46cd8f8955b8e.jpg

**Comment by danieldaquino** [comment]
@alltheseas, should we de-scope this from 1.8? (I think we talked about it on a meeting, but I just want to double-check)

**Comment by alltheseas** [comment]
> should we de-scope this from 1.8

Yes, this is my recommendation. We have many more important things to fix first, and I think it sends the wrong signal that we are not listening to customers who provide feedback that things are broken. 

Once we fix the top 5-6 things in [1.9](https://github.com/damus-io/damus/milestone/14) we can consider this one 🙏 .

**Comment by alltheseas** [comment]
> > should we de-scope this from 1.8
> 
> Yes, this is my recommendation. We have many more important things to fix first, and I think it sends the wrong signal that we are not listening to customers who provide feedback that things are broken.
> 
> Once we fix the top 5-6 things in [1.9](https://github.com/damus-io/damus/milestone/14) we can consider this one 🙏 .

Moving to 1.9 for now 

cc @jb55
